src/game_events/conditional_wml.hpp

```diff
@@ -34,5 +34,6 @@ namespace game_events
 		bool have_unit(const vconfig& cfg);
 		bool have_location(const vconfig& cfg);
 		bool variable_matches(const vconfig& cfg);
+		bool has_achievement(const vconfig& cfg);
```

Comment:
Now would be a good time to sort these into alphabetic order.

Reply:
It's been moved to `wesnoth.achievements` in `game_lua_kernel.cpp` at celticminstrel's request.